docs/philosophy/outcomes/platform_adoption.md

```diff
@@ -48,50 +60,188 @@ Measures a team’s ability to onboard into a new platform, tool or pattern and
+
+    Expressed as a _timestamp_
 
 #### Formulas
 
-At<sub>1</sub> = count(aE)[t<sub>1</sub>]
+_Adoptions at Time $t_{1}$ ( $a^{t_{1}}$ )_
 
-At<sub>2</sub> = count(aE)[t<sub>2</sub>]
+:   The number of Adoption end events $a_{E}$ that have taken place as of timestamp $t_{1}$. Calculated as follows:
 
-AR = (At<sub>1</sub> - At<sub>2</sub> - 1) * 100
+    $$
+    a^{t_{1}} = count(aE)[t_{1}]
+    $$
+
+_Adoptions at Time $t_{2}$ ( $a^{t_{2}}$)_
+
+:   The number of adoption end events $a_{E}$ that have taken place as of timestamp $t_{2}$ where $t_{2} > t_{1}$. Calculated as follows:
+
+    $$
+    a^{t_{2}} = count(aE)[t_{2}]
+    $$
+
+**_Adoption Rate $AR$_**
+
+$$
+AR = (a^{t_{2}} - a^{t_{2}} - 1) * 100
+$$
 
 ### :material-account-arrow-down: __Retention Rate__
 
-Measures the sustainability of the platform and whether it continues to be used beyond initial onboarding
+So we've got a platform up and running, onboarding is really fast and we've been able to attract a bunch of new users. Now we need to ask a new set of questions. Is the platform meeting the needs of its users? Are people continually using it? Measuring _retention rate_ tells us whether users that have adopted our platform continue to use it over time to do valuable work. It is the yin to _adoption rate's_ yang.
+
+Use time since last adoption on a varying time scale to determine active users.
+
+To determine your retention rate, first identify the time frame you want to study
+Next, collect the number of existing customers at the start of the time period ($u_{S}$)
+Then find the number of total customers at the end of the time period ($u_{E}$)
+Finally, determine the number of new customers added within the time period ($u_{N}$)
+
+#### Data Points
+
+_Time range ( $T$ )_
+
+:   The range of time in which you want to evaluate retention. For example:
+
+    * One month
+    * 3 months
+    * 6 months
+
+    Using this time range, we also need to set three timestamps:
+
+    1. The end of the time period we are measuring; $T_{0}$ (typically $T_{0} = now()$)
+    2. The start time of the time period; $T_{1} = T_{0} - T$
+    3. The start time of the previous time period; $T_{2} = T_{0} - 2T$
+
+_Adoption events by user ( $a_{E}\{u\}$ )_
+
+:   The set of all adoption events for each user
+
+#### Formulas
+
+_Latest adoption event by user ( $la_{E}\{u\}$ )_
+
+:   Given the set of all adoption events $a_{E}\{u\}$ for a given user, we simply select the latest timestamp
+
+    $$
+    la_{E}\{u\} = max(a_{E}\{u\})
+    $$
+
+_Active users_
+
+:   Given the set of all _latest adoption events by user ( $la_{E}\{u\}$ )_ above and the timestamp of the start of the time period we are measuring ( $T_{1}$ ), we then evaluate a binary expression to determine whether the user is "active":
+
+    $$
+    la_{E}\{u\} > T_{1}
+    $$
+
+    We can then count the total _existing active users_ based on the above filter:
+
+    $$
+    u_{E} = count(la_{E}\{u\} > T_{1})
+    $$
```

Comment:
did not understand this. It appears to be always 1 or 0

Reply:
hmm, understood, I think would be better to write like

`$$ count({u : U \in S, la_{E}\{u\} > T_{1}}) $$`

where `S` is the System. what you think?